org.theia_ide.theia_ide.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: org.theia_ide.theia_ide
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+modules:
+  - shared-modules/libsecret/libsecret.json
+  - theia_ide.yaml
+command: run.sh
+finish-args:
+  - --allow=devel
+  - --device=dri
+  - --env=ELECTRON_TRASH=gio
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=home
+  - --filesystem=host-os:ro
+  - --filesystem=xdg-cache
+  - --filesystem=xdg-config
+  - --filesystem=xdg-data
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=system-bus
```

Comment:
why does it need system bus?

Reply:
It doesn't now, but I'm sure I saw Theia complaining about the lack of DBus at some time. Maybe it was before I introduced `--ozone-platform-hint=auto` ? I can't tell, but it's removable now.